stdlib/seq.ml

```diff
@@ -711,3 +711,10 @@ let to_dispenser xs =
 
 let rec ints i () =
   Cons (i, ints (i + 1))
+
+module Syntax = struct
+  let ( let* ) r f = flat_map f r
+  let ( and* ) = product
+  let ( let+ ) r f = map f r
```

Comment:
The `r`'s could have been `xs` but that's nitpicking.

Reply:
Thanks, fixed (with `l` instead of `r`).